test/fixtures/client-with-bare-imports.js

```diff
@@ -1,13 +1,8 @@
-/* eslint-disable import/no-unresolved */
-
-'use strict';
-
+/* eslint-disable no-undef */
+// @ts-nocheck
 import scrollIntoView from 'scroll-into-view-if-needed';
 
 const main = () => {
-    // eslint-disable-next-line no-undef
-    const node = document.getElementById('hero');
```

Comment:
Not that this is the part under test, but I think this missing line is what `eslint` is complaining about 😅